src/pages/billing/Project.js

```diff
@@ -66,10 +67,10 @@ const WorkspaceCardHeaders = memoWithName('WorkspaceCardHeaders', ({ needsStatus
 const ExpandedInfoRow = ({ title, details, errorMessage }) => {
   const expandedInfoStyles = {
     row: { display: 'flex', justifyContent: 'flex-start', alignItems: 'flex-start' },
-    title: { fontWeight: 600, width: '20%', padding: '0.5rem 1rem 0 2rem', height: '1rem' },
-    details: { width: '20%', marginTop: '0.5rem', height: '1rem', ...Style.noWrapEllipsis },
+    title: { fontWeight: 600, padding: '0.5rem 1rem 0 2rem', height: '1rem' },
+    details: { flexGrow: 1, marginTop: '0.5rem', height: '1rem', ...Style.noWrapEllipsis },
     errorMessage: {
-      padding: '0.5rem', width: '55%', backgroundColor: colors.light(0.3),
+      flexGrow: 2, padding: '0.5rem', backgroundColor: colors.light(0.3),
```

Comment:
If there happens to be an error (which is only in the GCP case), give the error field more extra room. It will generally be longer, I think.